fossunited/fossunited/doctype/grants_funding_directory/test_grants_funding_directory.py

```
@@ -1,8 +1,203 @@
+
+    headers = {
+        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64)",
+    }
+
+    # Try 0x0.st first
+    try:
+        with open(file_path, "rb") as f:
+            r = requests.post("https://0x0.st", files={"file": f}, headers=headers, timeout=20)
+        if r.status_code == 200 and r.text.startswith("https://0x0.st"):
+            return r.text.strip()
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)